src/main/java/cloud/fogbow/ras/core/plugins/interoperability/aws/securityrule/v2/AwsV2SecurityRuleUtils.java

```diff
@@ -112,6 +101,19 @@ private Address getAddress(String allocationId, Ec2Client client) throws FogbowE
         return address;
     }
 
+    protected SecurityGroup getGroupById(String groupId, Ec2Client client) throws FogbowException{
+        DescribeSecurityGroupsRequest request = DescribeSecurityGroupsRequest.builder().groupIds(groupId).build();
+        SecurityGroup group = null;
+
+        try {
+            group = client.describeSecurityGroups(request).securityGroups().get(FIRST_POSITION);
```

Comment:
It would not be more elegant to use the method 'next' of the list-iterator in the security groups class.